2021q4/portmgr.adoc

```diff
@@ -0,0 +1,32 @@
+=== Ports Collection
+
+Links: +
+link:https://www.FreeBSD.org/ports/[About FreeBSD Ports] URL:link:https://www.FreeBSD.org/ports/[https://www.FreeBSD.org/ports/] +
+link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[Contributing to Ports] URL: link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[https://docs.freebsd.org/en/articles/contributing/#ports-contributing] +
+link:http://portsmon.freebsd.org/[FreeBSD Ports Monitoring] URL: link:http://portsmon.freebsd.org/[http://portsmon.freebsd.org/] +
+link:https://www.freebsd.org/portmgr/[Ports Management Team] URL: link:https://www.freebsd.org/portmgr/[https://www.freebsd.org/portmgr/] +
+link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[Ports Tarball] URL: link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[http://ftp.freebsd.org/pub/FreeBSD/ports/ports/]
+
+Contact: René Ladan <portmgr-secretary@FreeBSD.org> +
+Contact: FreeBSD Ports Management Team <portmgr@FreeBSD.org>
+
+The Ports Management Team is responsible for overseeing the overall direction of the Ports Tree, building packages, and personnel matters.
+Below is what happened in the last quarter.
+
+We currently have 46,700 ports in the Ports Collection according to FreshPorts.
+There are currently 2,666 open ports PRs of which 611 are unassigned.
+This quarter saw 9,535 commits from 166 committers on the main branch and 644 commits from 62 committers on the quarterly branch.
+Compared to last quarter, this means a slight drop in the number of commits although more committers were active, and a slight increase in the number of open PRs.
+
+During the last quarter, we welcomed Dries Michiels (driesm@) and said goodbye to kuriyama@ and fjoe@.
+There was also a change in portmgr: adamw@ stepped down after five years of service and tcberner@ is now a full member of portmgr@.
+
+Three new USES were introduced:
+* magick to handle dependencies on ImageMagick
+* nodejs to provide support for NodeJS (with a new default version NODEJS=lts)
+* trigger to handle pkg triggers using the TRIGGERS variable
+The default version of PGSQL switched to 13.
+Furthermore, INSTALLS_ICONS has been replaced by a trigger on gtk-update-icon-cache, the macro is no longer functional.
```

Comment:
Comma splice before "the macro is no longer functional". Instead of a comma, use "and" or a semicolon.